UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -260,16 +260,39 @@ private async Task CreateAncillaryFiles(string batchId, string exchangeSetPath,
         }
 
         public async Task<bool> DownloadReadMeFile(string batchId, string exchangeSetRootPath, string correlationId)
```

Comment:
This method assumes success as long as no exceptions are thrown. Consider adding error handling to manage cases where the methods fail unexpectedly. 
Unlike the other method _DownloadReadMeFileFromFss_, which involves an API call and may return a success code of false.
Also add unit test case to cover the same.

Reply:
added exception handling logic